vispy/visuals/visual.py

```diff
@@ -30,6 +30,11 @@ class Visual(object):
     bounds_change : Event
         Emitted when the bounds of the visual have changed.
 
+    Notes
```

Comment:
How about something like "When used in the scenegraph, all Visual classes are mixed with vispy.scene.Node in order to implement . . ."